### PR TITLE
1.10.238 Release

### PR DESCRIPTION
**Pull request by mfikes**

### CODE REVIEW

File: content/news/2018-xx-yy-core-specs-alpha-reducible-sequence-generators-and-map-entries.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+This feature is still alpha, but we encourage you to give it a try and report any defects you might find!
+
+## Reducible Sequence Generators
+
+With this ClojureScript release, the results of `iterate`, `repeat` and `cycle` are now directly reducible. This brings some great work that Alex Miller did for Clojure a http://insideclojure.org/2015/01/18/reducible-generators/[few years ago] to ClojureScript. This means that you will get much better performance when reducing over the output of these functions. 
+
+Take, for example, a benchmark involving running `(transduce (take 64) + (iterate inc 0))` a total of 10,000 times when compiled with `:advanced` optimizations. You can try this benchmark on your machine, but we are seeing this run 4.5 times faster under V8 and SpiderMonkey, and 3.3 times faster on JavaScriptCore.
+
+In addition, this provides a way to process large output without involving intermediate sequence generation, thus bypassing the lack of locals-clearing and inevitable head-holding that occurs in CloureScript. This means you can now run programs like
```
Comment (thomasmulvaney) [inline content/news/2018-xx-yy-core-specs-alpha-reducible-sequence-generators-and-map-entries.adoc]: Looks like a typo at ClourScript.

**Comment by Deraen** [comment]
# Module processing improvements and Closure update

ClosureScript uses now latest Closure-compiler release, which includes loads of improvements for consuming Node Modules. In addition to Closure-compiler update, several changes where implemented on ClojureScript side. Some of the improvements include:

- CommonJS and ES6 modules can now require each other
- Closure now detects and is able to remove more UMD wrappers
- Node module support can be disabled by setting `:npm-deps` option to `false`, for cases where `node_modules` directory exists but should not be used

**Comment by mfikes** [comment]
Need to document `:stable-names`

**Comment by swannodette** [comment]
This looks great.

**Pull request merged**